reproduce.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+venv_dir=".anomaly"
+if [ ! -d $venv_dir]
+then
+    python3 -m venv .anomaly
+    . .anomaly/bin/activate
```

Comment:
is `.` as supported as `source`?

Reply:
It works on ubuntu and manjaro but I took it out anyway